PhysicsTools/PatAlgos/test/patTuple_addBTagging_cfg.py

```diff
@@ -101,6 +101,17 @@
     ,'pfCombinedCvsBJetTags'
      # ChargeTagging
     ,'pfChargeBJetTags'
+     #Deep Flavour
+    ,'pfDeepCSVJetTags:probb'
+    ,'pfDeepCSVJetTags:probc'
```

Comment:
I guess you are aiming at the Phase I case only given the fact that there is no `probcc` here. This looks fine to me.